code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -1570,6 +1571,102 @@ func (srh *StorageRestHandler) getTransactionByFilter(w http.ResponseWriter, r *
 
 }
 
+// swagger:route GET /v1/screst/6dba10422e368813802877a85039d3985d96760ed844092319743fb3a76712d7/transactionHashes transactionHashes
+// Gets filtered list of transaction hashes from file information
+//
+// parameters:
+//    + name: look_up_hash
+//      description: restrict to transactions by the specific look up hash on write marker
+//      in: query
+//      type: string
+//    + name: name
+//      description: restrict to transactions by the specific file name on write marker
+//      in: query
+//      type: string
+//    + name: content_hash
```

Comment:
use hyphen `-` instead of underscore `_`, we will need to apply this for all URIs.